Core/clim-core/panes/stream-panes.lisp

```diff
@@ -365,49 +365,50 @@ current background message was set."))
 ;;;
 ;;; CONSTRUCTORS
 ;;;
-
 (defun make-clim-stream-pane (&rest options
                               &key (type 'clim-stream-pane)
+                                label
+                                label-alignment
+                                (display-after-commands nil display-after-commands-p)
```

Comment:
Please add `(check-type display-after-commands (member nil t :no-clear))` in case the option is supplied.

Reply:
done